libevm/ExtVMFace.h

```diff
@@ -169,6 +169,33 @@ class EnvInfo
     u256 m_gasUsed;
 };
 
+/// Represents a call result.
+///
+/// @todo: Replace with evmc_result in future.
+struct CallResult
+{
+    evmc_status_code status = EVMC_INTERNAL_ERROR;
+    owning_bytes_ref output;
+
+    CallResult(evmc_status_code status, owning_bytes_ref&& output)
+      : status{status}, output{std::move(output)}
+    {}
+};
+
+/// Represents a CREATE result.
+///
+/// @todo: Replace with evmc_result in future.
+struct CreateResult
+{
+    evmc_status_code status = EVMC_INTERNAL_ERROR;
+    h160 address;
+    owning_bytes_ref output;
+
+    CreateResult(evmc_status_code status, h160 address, owning_bytes_ref&& output)
```

Comment:
Better pass 20 bytes of `h160` by const reference

Reply:
I think I wanted to use `std::move` here. But for address it does not make sense anyway.